Templates/DefaultComponent/Template/Include/${Name}Interface.h

```diff
@@ -0,0 +1,48 @@
+// {BEGIN_LICENSE}
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+// {END_LICENSE}
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Component/EntityId.h>
+
+namespace ${SanitizedCppName}
+{
+    class ${SanitizedCppName}Requests
+    {
+    public:
+        AZ_RTTI(${SanitizedCppName}Requests, "{${Random_Uuid}}");
+        virtual ~${SanitizedCppName}Requests() = default;
+        // Put your public methods here
+
+        // Put notification events here
+        // void RegisterEvent(AZ::EventHandler<...> notifyHandler);
+        // AZ::Event<...> m_notifyEvent1;
```

Comment:
Are newer components more commonly using events or bus bound functions? I'm not familiar with the support for events and behavior context bindings. If the intention is to support both then I would include other examples in the comment.

Reply:
AZ Events are supported in the Behavior Context. It seems the documentation hasn't been published on o3de.org yet, but I can point you to it.